definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -6726,7 +6684,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "boAddTranslations",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
+    "CRUD": "R"
```

Comment:
and this too?

Reply:
Update: 67fd1ae81a33cefecd64d189c0c637ad893761be